pkg/apis/componentconfig/edgecore/v1alpha1/default.go

```diff
@@ -139,8 +139,12 @@ func NewDefaultEdgeCoreConfig() *EdgeCoreConfig {
 				ContextSendModule:  metaconfig.ModuleNameEdgeHub,
 				RemoteQueryTimeout: constants.DefaultRemoteQueryTimeout,
 				MetaServer: &MetaServer{
-					Enable: false,
-					Server: constants.DefaultMetaServerAddr,
+					Enable:            false,
+					AutonomySwitch:    false,
```

Comment:
Sugggest The default value set to true to compatible with previous version usage

Reply:
One reason is that we should make security by default, it is also an implementation of security hardening.